apps/emqx_machine/src/emqx_machine.erl

```diff
@@ -54,3 +55,15 @@ print_otp_version_warning() ->
     ?ULOG("WARNING: Running on Erlang/OTP version ~p. Recommended: 23~n",
           [?OTP_RELEASE]).
 -endif. % OTP_RELEASE > 22
+
+start_sysmon() ->
+    case application:get_env(system_monitor, db_hostname) of
+        undefined ->
```

Comment:
Nit: would it make sense to add a `enable :: boolean()` key to the config and check that?

Reply:
I was thinking about this too, but right now it doesn't make much sense enable this app unless the postgres sink is properly configured, as it's just going to hog the resources. So I left it out to simplify things.